config/locales/server.en.yml

```diff
@@ -5484,7 +5484,7 @@ en:
                 label: "Category boxes"
                 description: "Displays the categories and their description in a grid, ideal for members to see an overview of the sub-communities of your site"
 
-      logo_and_fonts:
```

Comment:
This was accidentally renamed when we changed the admin section from branding. It needs to match the wizard step for it to work.